templates/zerver/integrations.html

```diff
@@ -22,11 +22,21 @@
 
     <div class="portico-page-header"><a href="#"><i class="icon-vector-gears portico-page-header-icon"></i>Integrations</a></div>
 
-    <p class="portico-large-text">With Zulip integrations, your team can stay up-to-date on
-    code changes, issue tickets, build system results, and much more. If you don't see the system you would like to integrate with it, or run into any
-    trouble, don't hesitate to <a href="mailto:zulip-devel@googlegroups.com?subject=Integration%20question">email us</a>.</p>
-
-    <p>Many of these integrations require creating a Zulip bot. You can do so on your <a href="/#settings">Zulip settings page</a>. Be sure to note its username and API key.</p>
+    <p class="portico-large-text">With Zulip integrations, your team
+    can stay up-to-date on code changes, issue tickets, build system
+    results, tweets about your project, and much more.</p>
+
+    <p>If an integration you want isn't available yet, you
+    can <a href="http://zulip.readthedocs.org/en/latest/integration-guide.html">contribute
+    an integration</a>
+    or <a href="https://github.com/zulip/zulip/issues/new">open an
+    issue on GitHub</a> and tag it with the "integrations" label.  For
+    products that support webhooks, it often only takes a few hours to
```

Comment:
Good point, I removed that and also explained a benefit of opening an issue is so you can find out when the integration is done.